_articles/upgrade-pop.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: article
+title: Upgrade to Pop!_OS 18.04 LTS
```

Comment:
This should be:

Pop - Upgrade Pop!_OS

It doesn't need to be to a specific version, and we have a single word that leads the title.